Neos.Neos/Classes/FrontendRouting/NodeAddress.php

```diff
@@ -42,8 +32,9 @@
     /**
      * @internal use NodeAddressFactory, if you want to create a NodeAddress
      */
+    /** @phpstan-ignore-next-line its all just temporary */
     public function __construct(
-        public ContentStreamId $contentStreamId,
+        ?ContentStreamId $_contentStreamId,
```

Comment:
I assume, you just kept this to keep the PR smaller? But is it necessary to rename the parameter?

Reply:
its just a dummy stub, and `_` indicates its unused. the class will be obliterated into nothing.